overview/introduction.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+[TiKV](https://github.com/tikv/tikv) is a distributed, transactional key-value database. It has been widely used in the world, especially in many critical production environments. It has also been adopted by CNCF as a [sandbox project](https://www.cncf.io/blog/2018/08/28/cncf-to-host-tikv-in-the-sandbox/).
+
+TiKV provides a fully [ACID](https://en.wikipedia.org/wiki/ACID_(computer_science)) support, automatic horizontal scalability, global data consistency, Geo-replication, and many other useful features, it can be used as a building block to build other high-level services. E.g, we have already used TiKV to support [TiDB](https://github.com/pingcap/tidb) - a next-generation [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing_(HTAP)) database. 
```

Comment:
"provides a fully ACID support" sounds awkward. Maybe "TiKV is fully ACID while supporting automatic.,."